### ISSUE TITLE
Can't render constant graphs

### ISSUE THREAD

**Issue by felipeek** [wcharczuk/go-chart#92]
If the graph is constant (all Y values are the same) the constant line is not rendered

**Comment by quillaja** [comment]
YES! Oh god this is dumb... "zero y-range delta" is the error I'm getting. Sometimes the data is flat!

**Comment by jiuhuche120** [comment]
yeah! I think use max value of YValues is OK , same like `graph.YAxis.Range = &chart.ContinuousRange{Min: 0, Max: mathutil.Max()}`. It's better by default.
